docs/CHANGELOG.md

```diff
@@ -35,6 +35,7 @@ See also [LTS releases](https://docs.victoriametrics.com/lts-releases/).
 
 * SECURITY: upgrade base docker image (Alpine) from 3.20.1 to 3.20.2. See [alpine 3.20.2 release notes](https://alpinelinux.org/posts/Alpine-3.20.2-released.html).
 
+* FEATURE: `vmstorage` in the [Single-node VictoriaMetrics](https://docs.victoriametrics.com/) and the [VictoriaMetrics cluster](https://docs.victoriametrics.com/cluster-victoriametrics/): Refactor the code located in the `MustAddRows` function that looks for the corresponding partition when data is inserted. The new code efficiently determines whether all data to be inserted belongs to the corresponding partition by obtaining the maximum and minimum timestamps of all data to be inserted, resulting in better performance and readability.
```

Comment:
Those code changes look good to me.

```suggestion
* FEATURE: [VictoriaMetrics Single-Node](https://docs.victoriametrics.com/) and [VictoriaMetrics Cluster](https://docs.victoriametrics.com/cluster-victoriametrics/): Refactor the code located in the `MustAddRows` function that looks for the corresponding partition when data is inserted. The new code efficiently determines whether all data to be inserted belongs to the corresponding partition by obtaining the maximum and minimum timestamps of all data to be inserted, resulting in better performance and readability.
```

per some discussion in internal channel:
- VictoriaMetrics Single-Node (or Single Node)
- VictoriaMetrics Cluster